packages/manager/package.json

```
@@ -85,6 +85,7 @@
     "start:ci": "yarn serve ./build -p 3000 -s --cors",
     "lint": "yarn run eslint . --ext .js,.ts,.tsx --quiet",
     "build": "node scripts/prebuild.mjs && vite build",
+    "build:analyze": "npx vite-bundle-visualizer",
```

Optional: Use `bunx` if we're feeling crazy 🤪

```suggestion
    "build:analyze": "bunx vite-bundle-visualizer",
```

---

you got it 👍